From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Martijn Muijsers <martijnmuijsers@live.nl>
Date: Thu, 13 Jul 2023 08:42:26 +0200
Subject: [PATCH] Modifiable Bukkit enums - Remove hard-coded properties -
 Material - Name and key conversion utility

Adds the MaterialNameAndKeyConversion class (empty, to be populated in following commits).

License: Fiddle Public License Agreement 1.0 (included in license/FPL-1.0.txt)
Fiddle - https://fiddlemc.org

diff --git a/src/main/java/org/fiddlemc/fiddle/material/MaterialNameAndKeyConversion.java b/src/main/java/org/fiddlemc/fiddle/material/MaterialNameAndKeyConversion.java
new file mode 100644
index 0000000000000000000000000000000000000000..db0b953c6aacd29817dcb0bd5069346ac7116b80
--- /dev/null
+++ b/src/main/java/org/fiddlemc/fiddle/material/MaterialNameAndKeyConversion.java
@@ -0,0 +1,20 @@
+// Fiddle start - modifiable Bukkit enums - Material - remove hard-coded properties - name and key conversion
+
+package org.fiddlemc.fiddle.material;
+
+import org.bukkit.Material;
+
+/**
+ * This class provides a number of implementations for the conversion of names and keys
+ * from and to {@link Material} instances.
+ *
+ * @deprecated Do not use: for internal use only.
+ */
+@Deprecated(forRemoval = true)
+public final class MaterialNameAndKeyConversion {
+
+    private MaterialNameAndKeyConversion() {
+        throw new RuntimeException();
+    }
+
+}
